plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1208,10 +1206,6 @@ chain_apis::read_only chain_plugin::get_read_only_api(const fc::microseconds& ht
 }
 
 
-bool chain_plugin::accept_block(const signed_block_ptr& block, const block_id_type& id, const block_handle& bh ) {
```

Comment:
Like removing this indirection. No side effects by the removal?

Reply:
No, `net_plugin` already depends on `producer_plugin`, so it can call it directly.